bundles/org.openhab.binding.tradfri/src/main/java/org/openhab/binding/tradfri/internal/discovery/TradfriDiscoveryService.java

```diff
@@ -43,26 +48,28 @@
  * @author Kai Kreuzer - Initial contribution
  * @author Christoph Weitkamp - Added support for remote controller and motion sensor devices (read-only battery level)
  * @author Andre Fuechsel - fixed the results removal
+ * @author Manuel Raffel - Added support for blinds
  */
 @NonNullByDefault
-public class TradfriDiscoveryService extends AbstractDiscoveryService implements DeviceUpdateListener {
+public class TradfriDiscoveryService extends AbstractDiscoveryService
+        implements DeviceUpdateListener, DiscoveryService, ThingHandlerService {
     private final Logger logger = LoggerFactory.getLogger(TradfriDiscoveryService.class);
 
-    private final TradfriGatewayHandler handler;
+    private @Nullable TradfriGatewayHandler handler;
 
     private static final String REMOTE_CONTROLLER_MODEL = "TRADFRI remote control";
 
-    private static final String[] COLOR_TEMP_MODELS = new String[] { "TRADFRI bulb E27 WS opal 980lm",
-            "TRADFRI bulb E27 WS clear 950lm", "TRADFRI bulb GU10 WS 400lm", "TRADFRI bulb E14 WS opal 400lm",
-            "FLOALT panel WS 30x30", "FLOALT panel WS 60x60", "FLOALT panel WS 30x90",
-            "TRADFRI bulb E12 WS opal 400lm" };
+    private static final List<String> COLOR_TEMP_MODELS = Collections
```

Comment:
Why not use a `Set` here?

Reply:
Yes, good point. Why not use a `Set` here. I changed it.